OneLog.php

```diff
@@ -12,35 +12,13 @@
  *
  * @author Denis-Florin Rendler <connect@rendler.me>
  *
- * @method emergency()
- * @method static emergency()
- * @method alert()
- * @method static alert()
- * @method critical()
- * @method static critical()
- * @method error()
- * @method static error()
- * @method warning()
- * @method static warning()
- * @method notice()
- * @method static notice()
- * @method info()
- * @method static info()
- * @method debug()
- * @method static debug()
- * @method log()
- * @method static log()
- *
  * @property LoggerInterface $default
  */
 class OneLog
 {
-    public const DEFAULT_LOGGER = 'default';
+    use OneLogTrait;
```

Comment:
I would rename this to PSRLoggerTrait as it gives it a bit more context especially since it is a PSR\LoggerInterface implementation